tests/unit/stream_alert_shared/lookup_tables/test_driver_s3.py

```diff
@@ -181,15 +181,15 @@ def test_no_need_refresh(self, mock_logger):
     def test_barely_does_not_need_refresh(self, mock_logger):
```

Comment:
LOL at this test name `test_barely_does_not_need_refresh`

Reply:
_barely_